### DISCUSSION TITLE
PC Cold Start時の認識

### DISCUSSION THREAD

**Discussion by hiroyamagen** [Yowkees/keyball#494] [Q&A]
keyball39です
現状ですが、PC起動時にUSB接続した状態だと認識できないため、OS起動後にUSB抜き差しして認識させています
PC電源ON後、10秒程度経たないと認識できないようで、SPLIT_USB_TIMEOUTでも調整しきれません
毎回USB抜き差しは手間なのでなんとかしたいのですが、いい方法はないでしょうか？
常に左側にUSB接続します。右に接続することは(Firmware書き換え以外は)ありません

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。また、Keyball39をお選びいただき誠にありがとうございます。

PC起動時の挙動ですが、QMK_Firmware側の仕様でして、ご使用のPC環境に依存します。Firmwareのパラメータ調整でうまくいかないようですと、やはり再接続するしかないと思います。
マグネット接続タイプのUSBケーブルですと再接続が容易ですのでお試しいただければ幸いです。

このような回答で申し訳ありません。
お手数ですがご検討いただければ幸いです。

**Reply by hiroyamagen** [reply]
返信ありがとうございます
マグネット接続を使っているのですが、それでもコネクタがもげるのが心配だったので問い合わせていただきました。
もげないよう大事に使っていきたいと思います
